priv/graphql/schemas/admin/muc_light.gql

```diff
@@ -0,0 +1,43 @@
+"""
+Allow admin to manage Multi-User Chat Light rooms.
+"""
+type MUCLightAdminMutation @protected{
+  "Create a MUC light room under the given XMPP hostname"
+  createRoom(mucDomain: String!, name: String!, owner: JID!, subject: String!, id: String): Room
+  "Change configuration of a MUC Light room"
+  changeRoomConfiguration(room: JID!, owner: JID!, name: String!, subject: String!): Room
+  "Invite a user to a MUC Light room"
+  inviteUser(room: JID!, sender: JID!, recipient: JID!): String
+  "Remove a MUC Light room"
+  deleteRoom(room: JID!): String
+  "Kick a user from a MUC Light room"
+  kickUser(room: JID!, user: JID!): String
+  "Send a message to a MUC Light room"
+  sendMessageToRoom(room: JID!, from: JID!, body: String!): String
+}
+
+"""
+Allow admin to get information about Multi-User Chat Light rooms.
+"""
+type MUCLightAdminQuery @protected{
+  "Get the MUC Light room archived messages"
+  getRoomMessages(room: JID!, pageSize: Int!, before: DateTime): StanzasPayload
+  "Get configuration of the MUC Light room"
+  getRoomConfig(room: JID!): Room
+  "Get users list of given MUC Light room"
+  listRoomUsers(room: JID!): [RoomUser!]
+  "Get the list of MUC Light rooms that the user participates in"
+  listUserRooms(user: JID!): [JID!]
+}
+
+type Room{
+  jid: JID 
```

Comment:
Yes, it means that this field is nullable. I think I can add `!` because currently, we return the `Room` type always with the value present for each field.